frontend/tests/rp-connect.utils.ts

```diff
@@ -0,0 +1,106 @@
+import { expect, type Page, test } from '@playwright/test';
+
+export const createPipeline = async (
+  page: Page,
+  { pipelineName, description }: { pipelineName: string; description?: string },
+) => {
+  return await test.step('Create pipeline', async () => {
+    await page.goto('/rp-connect/create');
+    await page.getByTestId('pipelineName').fill(pipelineName);
+    if (description) {
+      await page.getByTestId('pipelineDescription').fill(description);
+    }
+    await page.getByRole('button', { name: 'Create' }).click();
+    // Wait for redirect to pipeline detail page
+    await page.waitForURL(`/rp-connect/${pipelineName}`);
+    await expect(page.getByText(pipelineName)).toBeVisible();
+  });
+};
+
+export const deletePipeline = async (page: Page, { pipelineName }: { pipelineName: string }) => {
+  return await test.step('Delete pipeline', async () => {
+    await page.goto('/connect-clusters');
+    await expect(page.getByRole('link', { name: pipelineName })).toBeVisible();
+    await page.getByTestId(`delete-pipeline-button-${pipelineName}`).click();
+    await page.getByTestId('delete-pipeline-confirm-button').click();
+    await expect(page.getByText('Pipeline Deleted')).toBeVisible();
+    await expect(page.getByRole('link', { name: pipelineName })).not.toBeVisible();
+  });
+};
+
+export const createPipelineViaWizard = async (
+  page: Page,
+  { pipelineName, inputType, outputType }: { pipelineName: string; inputType: string; outputType: string },
+) => {
+  return await test.step('Create pipeline via wizard', async () => {
+    await page.goto('/rp-connect/wizard');
+
+    // Step 1: Select input
```

Comment:
These could become their own `test.step()` as it helps with CI view where it fails. I usually found the smaller the step, the easier to dissect where the issue may be happening